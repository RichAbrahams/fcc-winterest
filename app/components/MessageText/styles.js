import styled from 'styled-components';

export const Text = styled.h1 `
    font-family: 'Joti One', cursive;
    color: white;
`;

