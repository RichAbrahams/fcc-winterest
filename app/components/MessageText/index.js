/**
*
* MessageText
*
*/

import React from 'react';
import { Text } from './styles';


function MessageText(props) {
  return (
    <Text>
      {props.children}
    </Text>
  );
}

MessageText.propTypes = {
  children: React.PropTypes.string,
};

export default MessageText;
