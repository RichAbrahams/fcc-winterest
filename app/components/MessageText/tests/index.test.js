import React from 'react';
import renderer from 'react-test-renderer';
import MessageText from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <MessageText />).toJSON();
  expect(tree).toMatchSnapshot();
});
