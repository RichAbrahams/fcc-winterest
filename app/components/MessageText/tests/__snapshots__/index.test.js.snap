exports[`test renders correctly 1`] = `
<h1
  className="kFQInW" />
`;
